[2025-07-14T15:29:07.126+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-14T15:29:07.152+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mon_premier_dag.task_2 manual__2025-07-14T15:29:04.968914+00:00 [queued]>
[2025-07-14T15:29:07.167+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mon_premier_dag.task_2 manual__2025-07-14T15:29:04.968914+00:00 [queued]>
[2025-07-14T15:29:07.195+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-07-14T15:29:07.215+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): task_2> on 2025-07-14 15:29:04.968914+00:00
[2025-07-14T15:29:07.225+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'mon_premier_dag', 'task_2', 'manual__2025-07-14T15:29:04.968914+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/mon_premier_dag.py', '--cfg-path', '/tmp/tmpee3t2w5l']
[2025-07-14T15:29:07.228+0000] {standard_task_runner.py:105} INFO - Job 22: Subtask task_2
[2025-07-14T15:29:07.230+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=818) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-14T15:29:07.231+0000] {standard_task_runner.py:72} INFO - Started process 819 to run task
[2025-07-14T15:29:07.288+0000] {task_command.py:467} INFO - Running <TaskInstance: mon_premier_dag.task_2 manual__2025-07-14T15:29:04.968914+00:00 [running]> on host 97ccc390339b
[2025-07-14T15:29:07.455+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mon_premier_dag' AIRFLOW_CTX_TASK_ID='task_2' AIRFLOW_CTX_EXECUTION_DATE='2025-07-14T15:29:04.968914+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-14T15:29:04.968914+00:00'
[2025-07-14T15:29:07.457+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-14T15:29:07.476+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-07-14T15:29:07.478+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "This is my first DAG!"']
[2025-07-14T15:29:07.492+0000] {subprocess.py:99} INFO - Output:
[2025-07-14T15:29:07.494+0000] {subprocess.py:106} INFO - This is my first DAG!
[2025-07-14T15:29:07.495+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-07-14T15:29:07.531+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-14T15:29:07.532+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=mon_premier_dag, task_id=task_2, run_id=manual__2025-07-14T15:29:04.968914+00:00, execution_date=20250714T152904, start_date=20250714T152907, end_date=20250714T152907
[2025-07-14T15:29:07.567+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-14T15:29:07.590+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-14T15:29:07.621+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
